woocommerce/google-listings-and-ads
# Display or hide attributes based on product type

### Changes proposed in this Pull Request:



Closes #570.

This PR adds the functionality to specify the applicable product types for each product attribute and allows attribute inputs to be displayed only on certain product types.

### Detailed test instructions:

1. Edit an existing product to add a new one
2. Choose any product type except simple and variable
3. The "Google Listings and Ads" tab should be hidden
4. Choose the simple product type
5. The tab should be displayed and contain inputs for the following attributes: GTIN, MPN, Brand, Gender
6. Update all of those attributes, save the product, and make sure all values are updated in DB (`postmeta` table)
7. Convert the product to a variable and make sure only the `Brand` attribute exists in the GLA tab along with the text from the screenshot on #570 
8. Go to the variations tab and edit the variations
9. Click on the Google Listings and Ads box on each variation form and make sure the following attributes have inputs there: GTIN, MPN, Gender
10. Update the attributes and make sure the values are updated in the `postmeta` table


### Changelog Note:

> Tweak - Hide/Display attributes based on the product type
